### PR TITLE
Reword Edge Developer Tools to Edge F12 DevTools

### PR DESCRIPTION
**Pull request by auchenberg**

### CODE REVIEW

File: f12-devtools-guide/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Meet the Microsoft Edge Developer Tools
+# Meet the Microsoft Edge F12 DevTools
```
Comment (brendyna) [inline f12-devtools-guide/README.md]: @auchenberg looks good--one question: should we have a space between "DevTools"?

**Comment by brendyna** [comment]
:+1:

File: f12-devtools-guide/README.md (hunk 1 above)
Comment (erikadoyle) [inline f12-devtools-guide/README.md]: @auchenberg @brendyna I'll leave it to you guys to merge this once you decide. In general, feel free to field PRs, though the cpub team is also happy to review and merge whenever you need another set of eyes.

Comment (abbycar) [inline f12-devtools-guide/README.md]: Going to merge and add the space between to align with the site. Will also update throughout the docs. Thanks!

**Pull request merged**